kuma/api/v1/tests/test_views.py

```diff
@@ -624,3 +624,24 @@ def test_stripe_subscription_canceled_sends_ga_tracking(
     track_event_mock_signals.assert_called_with(
         CATEGORY_MONTHLY_PAYMENTS, ACTION_SUBSCRIPTION_CANCELED, "webhook"
     )
+
+
+@mock.patch("kuma.users.newsletter.utils.check_is_in_sendinblue_list")
+@pytest.mark.django_db
+def test_sendinblue_unsubscribe(mock_check_sendinblue, client):
+    mock_check_sendinblue.return_value = False
+
+    email = "testuser@example.com"
+
+    user(
+        save=True, username="testuser", email=email, is_newsletter_subscribed=True,
+    )
+
+    response = client.post(
+        reverse("api.v1.sendinblue_hooks"),
+        content_type="application/json",
+        data={"event": "unsubscribe", "email": email},
+    )
+    assert response.status_code == 200
+
+    assert not User.objects.get(email=email).is_newsletter_subscribed
```

Comment:
First of all, `user` is a terrible name for a function. If you want to search-and-replace it with `create_user` I'd approve that. 
Either way, I think you should hold on to the User instance and replace this line here with:
```suggestion
    my_user.refresh_from_db()
    assert not my_user.is_newsletter_subscribed
```
It's a nit but it means you're absolutely certain it's the same object and not someone else.